vitest.config.ts

```diff
@@ -0,0 +1,39 @@
+/// <reference types="vitest" />
+import { defineConfig } from 'vite'
+import react from '@vitejs/plugin-react'
+import path from 'path'
+
+export default defineConfig({
+  plugins: [react()],
+  test: {
+    globals: true,
+    environment: 'happy-dom',
+    // setupFiles: ['./src/test/setup.ts'],
+    css: true,
+    // Include coverage configuration
+    coverage: {
+      provider: 'v8',
+      reporter: ['text', 'json', 'html'],
+      exclude: [
+        'node_modules/',
+        'src/test/',
+        '**/*.d.ts',
+        '**/*.config.*',
+        'src/vite-env.d.ts'
+      ]
+    },
+    // Mock WebAssembly for tests
+    pool: 'forks',
+    poolOptions: {
+      forks: {
+        singleFork: true
```

Comment:
this is super unfortunate to be setting already. this is gonna tank performance when running the tests locally. I know CI can often struggle running forked tests (we have to do something similar in coder/coder) but if it's just about that we should set this specifically in CI, not all the time.